src/components/common/QRCodeComponent.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+    <div class="">
```

Comment:
For easier searching in a markup I'd add 'c-qr-code-component' in a root element, I didn't follow the same approach everywhere though.